src/ingestor.ts

```diff
@@ -226,10 +226,15 @@ export default async function ingestor(req) {
         body.sig,
         receipt
       );
-    } catch (e) {
+    } catch (e: any) {
       if (typeof e !== 'string') {
         capture(e);
       }
+
+      if (e.errno === 1062) {
+        return Promise.reject('duplicate message');
+      }
+
```

Comment:
I think we should place it before capture, to avoid these errors on sentry. or we want to know them for now?

Reply:
PR is to be merged after the duplication check PR, after when, this error should not happen. We capture the error in case of.

Reply:
We don't need this anymore correct?

Reply:
I think this need #216

Reply:
Check removed, since should not happen anymore since merging of #216